10x/generate_demo_data/generate_demo_data.wdl

```diff
@@ -61,7 +61,7 @@ task ExtractIndicesSpecificChromosomeAlignments {
     File output_indices_json = "indices.json"
   }
   runtime {
-    docker: "ambrosejcarr/python3-scientific:0.2.0"
+    docker: "humancellatlas/python3-scientific:0.1.0"
```

Comment:
Since Ambrose had this at 0.2.0 already, it might make more sense to tag it 0.2.0 in humancellatlas as well and remove the 0.1.0 tag.

Reply:
I was treating the transfer as a reset, so I created the 0.1.0 version and propagated it throughout the wdls in a my uBam PR, but missed this wdl. We could make this change, but Rex's versioning will work and is at this point the path of minimal work. 

So: OK with it being incremented if you prefer, but want to be clear that I made this mistake (?) not Rex.

Reply:
@dshiga Personally speaking, I share the same feeling with @ambrosejcarr , I treated the docker versioning standardizing as a reset of tags. After all, we really need an Initial singularity on the timeline of our docker tags universe. 

The assumption I made here was: 
* For those images moved from other sources to HCA organization (like python3-scientific in this case), I reset it and tag it with `0.1.0`, which means **new, backward compatible functionality is introduced**, according to the definition of semantic versioning.
* For those images already existed in HCA organization, I tag the latest image with `0.1.0`, but at the same time, build and push a new image from the Dockerfile on Github to Docker Hub, if it is ahead of the existing image, increments it to, such as `0.1.1`.

So I don't think it is kind of a mistake, just which we want to follow 🤓

Reply:
It's kind of gross to go backwards in version numbers here, but I agree it's more practical to do that at this point, so 0.1.0 is fine.